### PR TITLE
Add metadata tables into doc #75

### PR DESCRIPTION
**Pull request by regislon**
# Prework

- [x] I understand and agree to this repository's [code of conduct](https://github.com/ltelab/disdrodb/blob/main/CODE_OF_CONDUCT.md).
- [x] I understand and agree to this repository's [contributing guidelines](https://github.com/ltelab/disdrodb/blob/main/CONTRIBUTING.md).
- [x] I have already submitted an [issue](https://github.com/ltelab/disdrodb/issues) or [discussion thread](https://github.com/ltelab/disdrodb/discussions) to discuss my idea with the maintainers.

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [x] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and communicate accordingly:

**The PR fulfills these requirements:**

- [x] It's submitted to the branch named as follow : 
  - Add a reader: `reader-<institute>-<campaign>`
  - Fix a bug: `bugfix-<some_key>-<word>`
  - Improve the doc: `doc-<some_key>-<word>`
  - Add a new feature: `feature-<some_key>-<word>`
  - Refactor some code: `refactor-<some_key>-<word>`
  - Optimize some code: `optimize-<some_key>-<word>`
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] Don't forget to link PR to issue if you are solving one.
- [x] All tests are passing.
- [x] New/updated tests are included

If adding a **new feature**, the PR's description includes:

- [x] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

# Related GitHub issues and pull requests

- Ref: #75

# Summary
Add metadata tables into doc.

### CODE REVIEW

**Review by ghiggi — DISMISSED** [review]

**Comment by ghiggi** [comment]
For me ready to merge I would say. 
I see that the csv are located in your fork: 
`:file: C:\projects\disdrodb-fork\docs\source\csv\Data Attribution.csv`
It'is maybe not worth to put it inside the repo within the doc folder somewhere? 
I let you merge when you think everything is set up ;)

File: docs/source/conf.py [hunk 1]
```diff
@@ -39,26 +40,74 @@
...
+
+
+# create rst text
+metadata_keys_text = """
+=========================
+Metadata keys
+=========================
+
 
+The following tables are based on `this google sheet <https://docs.google.com/spreadsheets/d/1z1bh55BFTwp7u-069PD8NF6r_ZmpCQwr7i78W6RBY_g>`__.
```
Comment (ghiggi) [inline docs/source/conf.py:67]: Here we provide a link that enable edit the metadata.
You should eventually here use a link that enable just a view mode @regislon  ;)

File: docs/source/metadata_keys.rst [hunk 2]
```diff
@@ -1,55 +1,75 @@
...
 
+| 
+Sensor Location
+===============
 
-Dimensions
-========================
+.. csv-table:: Sensor Location
+   :align: left
+   :file: C:\projects\disdrodb-fork\docs\source\csv\Sensor Location.csv
```
Comment (ghiggi) [inline docs/source/metadata_keys.rst:40]: We could add the csv to the repo maybe? Within the doc directory somewhere?

**Review by ghiggi — COMMENTED** [review]
For me the PR is almost ready to be merged. 
Just maybe resolve the comments that I indicated ... 
I let you merge when you think everything is solved ;)

File: docs/source/conf.py (hunk 1 above)
Comment (regislon) [reply docs/source/conf.py:67]: yes, but only the owner of the google sheet can do that @ghiggi.

Comment (ghiggi) [reply docs/source/conf.py:67]: Ok. I can't generate two types of links (edit and view mode). 
Maybe we could directly add the full csv in the docs? Or already the separated csv? @regislon ?

Comment (regislon) [reply docs/source/conf.py:67]: the csv is generated dynamically while builing the doc

File: docs/source/metadata_keys.rst (hunk 2 above)
Comment (regislon) [reply docs/source/metadata_keys.rst:40]: the csv is generated dynamically while builing the doc @regislon

**Review by ghiggi — APPROVED** [review]

**Pull request merged**